src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/AlertAddressForm.tsx

```diff
@@ -57,7 +57,7 @@ export const AlertAddressForm = (props: Props) => {
     : props.data.monthly_monitor_report_free;
 
   // TODO: Deprecate this when monthly report for free users has been created
-  const monthlyFreeUserReportEnabled =
+  const canToggleFreeOrPlusMonthlyReport =
     props.enabledFeatureFlags.includes("MonthlyReportFreeUser") ||
     hasPremium(props.user);
```

Comment:
Ah sorry, that's because I was confused, seeing as it also controlled showing the toggle for Plus users (where the old name implied it was only for the free report). Indeed we should be able to remove it once the `MonthlyReportFreeUser` flag is removed, and replace it with just the `hasPremium` call.